app/build.gradle

```diff
@@ -117,6 +117,8 @@ dependencies {
   implementation "androidx.room:room-runtime:2.2.3"
   androidTestImplementation "androidx.room:room-testing:2.2.3"
 
+  implementation "com.google.code.gson:gson:2.8.0"
```

Comment:
We already have a json converter: Moshi, are we using any feature that Moshi doesn't provide?